lighthouse-core/audits/predictive-perf.js

```diff
@@ -123,13 +126,13 @@ class PredictivePerf extends Audit {
       let sum = 0;
       const values = {};
       Object.keys(graphs).forEach(key => {
-        const estimate = PageDependencyGraph.estimateGraph(graphs[key]);
-        const lastLongTaskEnd = PredictivePerf.getLastLongTaskEndTime(estimate.nodeTiming);
+        const results = new Simulator(graphs[key]).simulate();
```

Comment:
I'd personally still prefer `estimate` for the local var here, as `results` is information-less :) or `simulatedResults` or something. `results` could be anything

Reply:
done